benchmark_v2/no_op_benchmark.py

```diff
@@ -0,0 +1,33 @@
+import arkouda as ak
+import numpy as np
+import pytest
+
+SECONDS = pytest.time_trial

```

Comment:
The original benchmark has `--trials` and `--trials-time` setting the same value. We should remove `pytest.time_trial` and just use `pytest.trials`.